src/Sylius/Bundle/ProductBundle/Behat/ProductContext.php

```diff
@@ -292,4 +292,35 @@ public function theFollowingOptionTranslationsExist(TableNode $table)
 
         $manager->flush();
     }
+
+    /**
+     * @Given there is a disabled product :name
+     */
+    public function thereIsDisabledProduct($name)
+    {
+        $this->thereIsProduct($name, false);
+    }
+
+    /**
+     * @Given there is an enabled product :name
+     */
+    public function thereIsEnabledProduct($name)
+    {
+        $this->thereIsProduct($name, true);
+    }
+
+    /**
+     * @Given /^There is a product named "([^""]*)"$/
```

Comment:
I've already had that idea, but I'm wondering if it is enough to only retrieve a Product from repository? Or should I leave that step and implement logic of creating new product if none with given name exists?